packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2284,7 +2284,8 @@ export const components = ({
       },
       styleOverrides: {
         select: {
-          paddingBlock: odysseyTokens.Spacing3,
+          height: "auto",
+          paddingBlock: `calc(${odysseyTokens.Spacing3} - 1px)`,
```

Comment:
I like it. Maybe a comment about why we are subtracting a pixel here would be helpful. Also... why are we subtracting a pixel here? ha

Reply:
I just added documentation! We're aiming for 40px for the total height, but the 1px borders push us over to 42px. I just modified this to use the border width token so it's clearer what's happening here